OOP/LibraryManagement/LibraryApp/Documents/Document.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Text.Json.Serialization;
+
+namespace LibraryApp.Documents;
+
+public abstract class Document
```

Comment:
I would advise to ad `Base` prefix for some base abstract classes, it helps to indication on programming level that this is something common and should be inherited from